src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -356,7 +356,10 @@ def remove_unused_lvm_pv
 
           return unless device.lvm_pv
 
-          device.remove_descendants if lvm_pv.descendants.empty?
+          device = blk_device.encryption || blk_device
+          lvm_pv = device.lvm_pv
+
+          device.remove_descendants if lvm_pv && lvm_pv.descendants.none?
```

Comment:
Be careful, not the method is defined as:

```
        def remove_unused_lvm_pv
          device = blk_device.encryption ? blk_device.encryption : blk_device
          lvm_pv = device.lvm_pv

          return unless device.lvm_pv

          device = blk_device.encryption || blk_device
          lvm_pv = device.lvm_pv

          device.remove_descendants if lvm_pv && lvm_pv.descendants.none?
        end
```

But my proposal was:

```
        def remove_unused_lvm_pv
          device = blk_device.encryption || blk_device
          lvm_pv = device.lvm_pv

          device.remove_descendants if lvm_pv && lvm_pv.descendants.none?
        end
``` 

Sorry, maybe my suggestion was misleading.

Reply:
No worries, I'll fix it.